src/document-creator/document-creator.mdx

```diff
@@ -43,23 +44,44 @@ The configuration file is a JSON file that contains information to configure the
 {
   "network": "ropsten",
   "wallet": "{\"address\":\"6a36c563a5350d7be66c801f901a67...\", ...}}",
-  "forms": [{...}]
+  "forms": [{...}],
+  "documentStorage": {
+    "apiKey": "kNb15YYZ6N1zBlYd25cjj8PLgK6YAuvN9Gf7fPM1",
+    "url": "https://api-ropsten.tradetrust.io/storage"
+  }
 }
 ```
 
-### `"network"` field
+Let's break down the required fields for this config file.
+
+We will need:
+
+- [Network field](#network-field)
+- [Wallet field](#wallet-field)
+- [Forms field](#forms-field)
+- [Document Storage field](#document-storage-field)
+
+## Network field
+
+The `"network"` field is a string that refers to the network in which the document is created in.
 
-The `network` field is a string that refers to the network in which the document is created in.
+As of now, we only cater to 3 networks.
 
-### `"wallet"` field
+- Mainnet : `"homestead"`
+- Ropsten testnet : `"ropsten"`
+- Rinkeby testnet : `"rinkeby"`
 
-The `wallet` field is a string that refers to your etherum wallet.
+## Wallet field
 
-To convert the wallet into a string like the above example, please use this [online tool](https://tools.knowledgewalls.com/jsontostring).
+The `"wallet"` field is a string that refers to your etherum wallet.
 
-### Forms structure
+If you have the wallet you can convert the wallet into a string like the above example, please use this [online tool](https://tools.knowledgewalls.com/jsontostring).
```

Comment:
i think we shouldn't be recommending people to copy and paste their wallets into random online forms 🐱
can just suggest people to open their browser console and paste in the wallet object and do JSON.stringify there if necessary or modify oa cli to have a flag for stringified output

(why is it a stringified json instead of just an object?)

Reply:
ok will remove the online form.

regarding your question im not sure, need ask raymond. when i was doing it with him its already a stringified json, i thought it was like that all along.